emulator/fp-emu

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
what is the purpose of this script; was there a problem with `fp-emu +ispm=program.mem`?

Reply:
Thanks, yes now that the ISPM is actually in verilog we can use the plusargs again. Previously the loading of ISPM was done in Chisel and it had to be hardcoded. I had actually forgot that this changed with the DualPortBram.V